src/stratigraphy/benchmark/score.py

```diff
@@ -198,27 +193,21 @@ def create_predictions_objects(
         ground_truth_is_present = False
 
     number_of_truth_values = {}
-    predictions_objects = {}
-    for file_name, file_predictions in predictions.items():
-        metadata = metadata_per_file.get_metadata(file_name)
-        if not metadata:
-            raise ValueError(f"Metadata for file {file_name} not found.")
+    for file_predictions in predictions.file_predictions_list:
+        # prediction_object = FilePredictions.create_from_json(file_predictions, file_predictions.file_name)
 
-        prediction_object = FilePredictions.create_from_json(file_predictions, metadata, file_name)
-
-        predictions_objects[file_name] = prediction_object
+        # predictions_objects[file_name] = prediction_object
```

Comment:
Thank you. I commented it out to test whether the new code works but did not remove the code in case I had to roll back. However, since this version seems functional the commented code is not needed anymore.